### PR TITLE
Add view for ETH transfers of Gnosis Safes

### PR DESCRIPTION
**Pull request by tschubotz**
I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/gnosis_safe/view_safes_eth_transfers.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    JOIN gnosis_safe.view_safes s
+        ON et."to" = s.address
+    WHERE success = TRUE
+        AND value <> 0
+        AND block_time > '2018-11-25'  -- There have been no Safe contracts before
+        AND (call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR call_type IS null)
+)
+
+CREATE UNIQUE INDEX CONCURRENTLY IF NOT EXISTS view_safes_eth_transfers_unique_idx ON gnosis_safe.view_safes_eth_transfers (tx_hash);
+CREATE INDEX IF NOT EXISTS view_safes_eth_transfers_block_time_idx ON dgnosis_safeex.view_safes_eth_transfers USING BRIN (block_time);
```
Comment (sche) [inline ethereum/gnosis_safe/view_safes_eth_transfers.sql:35]: Is it a type `dgnosis_safeex`?

Comment (tschubotz) [reply ethereum/gnosis_safe/view_safes_eth_transfers.sql]: yes, thanks!

File: ethereum/gnosis_safe/view_safes_eth_transfers.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
+BEGIN;
+DROP MATERIALIZED VIEW IF EXISTS gnosis_safe.view_safes_eth_transfers;
+
+CREATE MATERIALIZED VIEW gnosis_safe.view_safes AS
```
Comment (masquot) [inline ethereum/gnosis_safe/view_safes_eth_transfers.sql:4]: I would expect `view_safes_eth_transfers` here. Is that correct?

Comment (tschubotz) [reply ethereum/gnosis_safe/view_safes_eth_transfers.sql]: 🙈 Thank you. Fixed.

**Comment by tschubotz** [comment]
closing this since it might no be needed actually. will need to reevaluate

**Pull request closed without merging**